src/api-wrappers/AXUIElement.swift

```diff
@@ -247,6 +247,80 @@ extension AXUIElement {
     func performAction(_ action: String) {
         AXUIElementPerformAction(self, action as CFString)
     }
+    
+    private static let ignoredBundleIDs = Set([
+        "com.apple.dashboard",
+        "com.apple.loginwindow",
+        "com.apple.notificationcenterui",
+        "com.apple.wifi.WiFiAgent",
+        "com.apple.Spotlight",
+        "com.apple.systemuiserver",
+        "com.apple.dock",
+        "com.apple.AirPlayUIAgent",
+        "com.apple.dock.extra",
+        "com.apple.PowerChime",
+        "com.apple.WebKit.Networking",
+        "com.apple.WebKit.WebContent",
+        "com.apple.WebKit.GPU",
+        "com.apple.FollowUpUI",
+        "com.apple.controlcenter",
+        "com.apple.SoftwareUpdateNotificationManager",
+        "com.apple.TextInputMenuAgent",
+        "com.apple.TextInputSwitcher"
+    ])
+
+    /**
+     *  Returns a Bool indicating whether or not the application will have windows.
+     *
+     *  @return true if the application will have windows and false otherwise.
+     */
+    static func isManageable(_ runningApp: NSRunningApplication) -> Bool {
+        guard let bundleIdentifier = runningApp.bundleIdentifier else {
```

Comment:
we have cases like the android emulator where the app has no `bundleIdentifier` but is still app with windows (see https://github.com/lwouis/alt-tab-macos/blob/02590579df936d6a2124707e12d76865636fe23f/src/logic/Applications.swift#L127)